odoo/modules/module_graph.py

```diff
@@ -134,7 +134,7 @@ class ModuleNode:
     def __init__(self, name: str, module_graph: ModuleGraph) -> None:
         # manifest data
         self.name: str = name
-        self.manifest: dict = get_manifest(name)
+        self.manifest: dict = _get_manifest_cached(name)  # the cached value directly to and to avoid deepcopy for performance
```

Comment:
It should be safe to use the not deepcopy version. Since the data is only used for loading registry

maybe `module.write(module.get_values_from_terp(package.manifest))` in `load_module_graph` can be risky when `update_module=True`, then we can still deepcopy the manifest

Reply:
Ok, we can leave as is.

Reply:
The real question is this: is the manifest safe there?  If `ModuleGraph` does not modify the manifest dicts, and does not spread them around, then they are safe.